app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -89,7 +99,9 @@ class DrawActivity : AppCompatActivity() {
                     onDialogComplete = { viewModel.dialogCompleted() },
                     onDelete = { index ->
                         viewModel.deleteLayer(index)
-                        bottomSheetLambda?.invoke(index, null, DialogOperation.DELETE)
+                        if (!viewModel.isLayerListSize()) {
```

Comment:
In this case you could toast that they can't delete the last layer or they won't know why? Or is that elsewhere?

Reply:
Should toast when it invokes if it sees that the list is empty, this lambda does need a kdoc of some sort so that its clear what it does. Even I had to dig a bit to relearn what it did here so I added that.

The toast will be sent in the function to delete a layer if the list only has one entry left.